tests/functional/Agent.php

```diff
@@ -500,5 +500,124 @@ public function testStaleActions()
         $this->boolean($item->getFromDB($item->fields['id']))->isTrue();
         $this->integer($item->fields['is_deleted'])->isIdenticalTo(1);
         $this->integer($item->fields['states_id'])->isIdenticalTo($states_id);
+
+        //create another new status
+        $states_id2 = $state->add(['name' => 'Stale2']);
+        $this->integer($states_id2)->isGreaterThan(0);
```

Comment:
You can use the `createItem` method to simplify your tests:

```suggestion
        $state2 = $this->createItem(\State::class, ['name' => 'Stale2']);
        $states_id2 = $state2->getID();
```

It do the `->isGreaterThan(0)` part for you and will also run additional tests to ensure all supplied values were set correctly.

Reply:
fixed